diff --git a/dist/preset.js b/dist/preset.js
--- a/dist/preset.js
+++ b/dist/preset.js
@@ -187,7 +187,7 @@
 ), pathToFileURL(args[0]).href) : import.meta.resolve(...args);
 
 // src/node/boot-test-runner.ts
-var MAX_START_TIME = 3e4, vitestModulePath = fileURLToPath2(importMetaResolve("@storybook/addon-vitest/vitest")), eventQueue = [], child, ready = !1, unsubscribeStore, unsubscribeStatusStore, unsubscribeTestProviderStore, forwardUniversalStoreEvent = (storeEventName) => (event, eventInfo) => {
+var MAX_START_TIME = 12e4, vitestModulePath = fileURLToPath2(importMetaResolve("@storybook/addon-vitest/vitest")), eventQueue = [], child, ready = !1, unsubscribeStore, unsubscribeStatusStore, unsubscribeTestProviderStore, forwardUniversalStoreEvent = (storeEventName) => (event, eventInfo) => {
   child?.send({
     type: storeEventName,
     args: [{ event, eventInfo }],
